predictor/framework/resource.cpp

```diff
@@ -36,7 +36,13 @@ DynamicResource::DynamicResource() {}
 
 DynamicResource::~DynamicResource() {}
 
-int DynamicResource::initialize() { return 0; }
+int DynamicResource::initialize() { 
+    return 0;
```

Comment:
Indent with 2 spaces

Reply:
have been auto-fix by pre-commit, thank you.